<?xml version="1.0"?>
<launch>
	<arg name="car_id"/>
	<arg name="speed" default="0.1"/>
	<arg name="map" default="cimat_reduced"/>
	<arg name="look_ahead" default="30cm"/>
	<arg name="lane" default="1"/>

	<remap from="/sensors/obstacles" to="/v2x/sensors/obstacles/filtered"/>

	<node name="cooperative_convoy_$(arg car_id)" pkg="cooperative_driving" type="cooperative_convoy_node.py">
		<param name="car_id" value="$(arg car_id)"/>
		<param name="speed" value="$(arg speed)"/>
		<param name="map_name" value="$(arg map)"/>
		<param name="look_ahead" value="$(arg look_ahead)"/>
		<param name="lane" value="$(arg lane)"/>
	</node>
</launch>